monitor/census.go

```diff
@@ -554,13 +590,27 @@ func InitCensus(nodeType NodeType, version string) {
 			TagKeys:     append([]tag.Key{census.kOrchestratorURI}, baseTags...),
 			Aggregation: view.Distribution(0, .10, .20, .50, .100, .150, .200, .500, .1000, .5000, 10.000),
 		},
+		{
+			Name:        "upload_time_seconds_per_stream",
+			Measure:     census.mUploadTime,
+			Description: "UploadTime, seconds",
+			TagKeys:     append([]tag.Key{census.kOrchestratorURI}, baseTagsWithManifestID...),
```

Comment:
Did you mean only `baseTagsWithManifestID` here?

If we also have `kOrchestratorURI` this means that we can derive `upload_time_seconds` from this `upload_time_seconds_per_stream`, which means that we don't need to create the separate metric here.

Edit: Just saw now that this is using a single bucket distribution instead. This is the main thing I am confused about this PR which is repeated on many metrics, I'll stop commenting on the reoccurencies now. Is this the OpenCensus way of creating a [summary](https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations)?

Reply:
Ah I see! And what is exactly this single bucket distribution? Is it, in practice, another `count`?

Nice trick to avoid creating separate `sum` and` count` metrics btw :)